front/app/containers/Authentication/useSteps/index.ts

```diff
@@ -186,13 +192,7 @@ export default function useSteps() {
           context: GLOBAL_CONTEXT,
         };
 
-        transition(
-          currentStep,
-          'START_INVITE_FLOW'
-        )(search).catch(() => {
-          setCurrentStep('sign-up:invite');
-          setError('invitation_error');
-        });
+        transition(currentStep, 'START_INVITE_FLOW')(search);
       }
```

Comment:
Just wondering - was the catch never being hit? Or why were we able to remove it here? :)

Reply:
This is now all handled in `useSteps/stepConfig/sharedSteps.ts`. This way `setCurrentStep` is only used inside of the state machine- seemed cleaner to me